src/components/Dashboard/Dashboard.js

```diff
@@ -49,7 +49,7 @@ export class Dashboard extends Component {
 
             this.setHeaderbarVisibility()
 
-            this.scrollToTop()
+            document.querySelector('.dashboard-wrapper')?.scroll(0, 0)
```

Comment:
To do this correctly, this should probably happen in the route handler rather than when clicking the dashboard link - that would solve the problem @edoardo saw

Reply:
Actually, moving it further down to just the View dashboard in an effect, seems to do the trick.